resources/install.rb

```diff
@@ -79,7 +80,7 @@
     make_cmd << " USE_LINUX_TPROXY=#{use_linux_tproxy}"
     make_cmd << " USE_LINUX_SPLICE=#{use_linux_splice}"
 
-    extra_cmd = ' EXTRA=haproxy-systemd-wrapper' if node['init_package'] == 'systemd'
+    extra_cmd = ' EXTRA=haproxy-systemd-wrapper' if node['init_package'] == 'systemd' && !install_only
```

Comment:
I'm sorry, I'm not sure what the change is being requested here. Is this in regards to `node['init_package']`? That's existing code, and as far as I can tell should still be supported in Chef 13, but I could be wrong on that and I'm happy to make that change.

That will also need to change on line 123 if that's the case, and I'd rather make a new PR for it.

If that's not the change requested then I really don't understand.